ubuntu

```diff
@@ -3,6 +3,8 @@ FROM ubuntu:latest
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get -q update -y
 RUN apt-get -qq install -y sudo make cmake automake autoconf libtool cmake-data git vim g++ gfortran flex doxygen graphviz pandoc python2.7 libopenmpi-dev libhdf5-openmpi-dev libcereal-dev liblapacke-dev libexodusii-dev libscotch-dev libmetis-dev python-pip python3-pip texlive-font-utils clang-7 clang-format-7 llvm-7 ccache texlive-latex-base texlive-fonts-recommended texlive-latex-recommended gdb wget curl lcov libboost-all-dev libclang-7-dev llvm-7-dev
+RUN ln -s /usr/bin/clang-7 /usr/bin/clang
```

Comment:
Wouldn't `update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-7 100` be the better option?

Reply:
sure